gprofiler/profilers/python.py

```diff
@@ -496,7 +511,7 @@ def stop(self) -> None:
     # we build pyspy for both, pyperf only for x86_64.
     # TODO: this inconsistency shows that py-spy and pyperf should have different Profiler classes,
     # we should split them in the future.
-    supported_archs=["x86_64", "aarch64"],
+    supported_archs=["x86_64", "aarch64", "AMD64"],
```

Comment:
`AMD64` and `x86_64` are the same, is this because Windows reports `AMD64`?

Reply:
Yes, Windows reports AMD64.

Reply:
I think it makes more sense here, to keep `supported_archs` for Linux, and add `supported_windows_archs` for the Windows support.

Then, for py-spy, we add `supported_windows_archs=["AMD64"]`. For all other profilers, we don't add, and thus only the Python one can be used now. Then we just have to fix the code in `get_profilers` to use `supported_windows_archs` when appropriate, i.e on Windows, otherwise the old `supported_archs`.

Reply:
Done